### DISCUSSION TITLE
WT32-SC01 Plus Touch Compatibility

### DISCUSSION THREAD

**Discussion by greysneakthief** [arendst/Tasmota#21820] [Support request]
Looking to get touch compatibility working with a WT32-SC01 Plus. As per [this prior discussion](https://github.com/arendst/Tasmota/discussions/17858), the final comments end on a cliffhanger.

The display itself works fine with the proper configuration from the Tasmota repository. I've not been able to get the touch part of the display to work. I've even uploaded a version of OpenHASP to test if it's not a hardware issue, but to my surprise it worked out of the box with minimal configuration. So now I'm wondering if I'm not doing something properly for Tasmota in the configuration.

Using [this .ini file](https://github.com/arendst/Tasmota/blob/development/tasmota/displaydesc/WT32_SC01_Plus.ini).
And this config: {"NAME":"WT32-SC01-Plus","GPIO":[0,1,1,1,0,0,0,1,1,1,1376,1,5408,1,1,1,1,1,1,1,1,1,0,0,0,0,0,6210,0,0,0,0,1,1,0,0,0,0],"FLAG":0,"BASE":1}
Universal display driver is enabled, and I'm using the latest version of Tasmota lvgl-haspmota.

**Comment by sfromis** [comment]
Haven't tried that display, but by now I think that you need to use the "full" touch driver definition in the `display.ini` file, to config for the specific touch controller IC on the cable.
https://tasmota.github.io/docs/Universal-Display-Driver/#universal-touch-driver-utouch

My *guess* would be FT5206, but that's not something I can vouch for. What does the square black chip on the small ribbon cable say?

**Reply by greysneakthief** [reply]
The chip is an FT6336U. I'll try to craft a full touch def for it then. Not super savvy with that but I'll consider it a learning experience.
